doc/ref_model/chapters/chapter07.md

```diff
@@ -339,6 +339,13 @@ A good place to start to understand the requirements is to use the widely accept
 Additional Cloud Infrastructure security principles that need to be incorporated:
 * Authenticity – The ability to confirm the users are in fact valid users with the correct rights to access the systems or data.
 
+In mobile network field, the GSM Association ([GSMA](https://www.gsma.com/)) and its Fraud and Security working group developed a strong expertise to secure the global mobile ecosystem. 
+* The document “Baseline Security controls”, [FS.31 v2.0](https://www.gsma.com/security/resources/fs-31-gsma-baseline-security-controls/), published in February 2020, is a practical guide intended for operators and stakeholders to check mobile network’s internal security. It lists a set of security controls from business controls (including security roles, organizational policies, business continuity management…) to technological controls (for user equipment, networks, operations…) covering all areas of mobile network, including Cloud Infrastructure. A checklist of questions allows to improve the security of a deployed network. 
+
+GSMA security activities are currently intense around 5G services and the new challenges raised by network functions virtualisation and open source software. The 2 following documents are in the scope of Cloud Infrastructure security:
+* The white paper [“Open Networking & the Security of Open Source Software deployment”](https://www.gsma.com/futurenetworks/resources/open-networking-the-security-of-open-source-software-deployment/), published in January 2021, deals with open source software security, it highlights the importance of layered security defences and lists recommendations and security concepts able to secure deployments. 
+* The “5G security Guide”, FS.40 version 1.0, Sept. 2020 (GSMA members only) is covering 5G security in a holistic way from user equipment to networks. The document describes the new security features in 5G. It includes a dedicated section on the impact of Cloud on 5G security with recommendations on virtualization, cloud native applications and containerization security. 
```

Comment:
Typo: “5G security Guide”  ---> “5G Security Guide”

In line with tense in other sentences suggest changing "...  is covering 5G security in a holistic way ..." to "...  covers 5G security, in a holistic way, ..." (including add of two commas)

Reply:
@Pankaj, thanks, done